### PR TITLE
Update Service Fabric SDK to version 6.0.1107

### PR DESCRIPTION
**Pull request by waynemunro**

### CODE REVIEW

File: src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <IncludeSymbols>true</IncludeSymbols>
-    <Platforms>x64</Platforms>
```
Comment (K-Cully) [inline src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj:24]: The platform changes appear to be out of scope.

**Review by neilr81 — APPROVED** [review]

File: test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <Description>ServiceFabric.Mocks contains Mock classes to enable unit testing of Actors and Services</Description>
     <Copyright>2022</Copyright>
     <AssemblyTitle>ServiceFabric.Mocks.NetCoreTests</AssemblyTitle>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```
Comment (loekd) [inline test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj:7]: Please revert this, we're not versioning the test dll.

File: src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj [hunk 3]
```diff
@@ -21,7 +21,7 @@
...
+    <Platforms>AnyCPU</Platforms>
```
Comment (loekd) [inline src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj:24]: Service Fabric is x64 only, so please revert.

File: upgrade-assistant.clef [hunk 4]
```diff
@@ -0,0 +1,3605 @@
+{"@t":"2022-09-22T17:33:37.5088235Z","@mt":"Hosting starting","@l":"Debug","EventId":{"Id":1,"Name":"Starting"},"SourceContext":"Microsoft.Extensions.Hosting.Internal.Host"}
```
Comment (loekd) [inline upgrade-assistant.clef:1]: Please delete this file

**Comment by loekd** [comment]
Sorry about the late response. Please resolve the open issues, so I can complete the PR.

File: test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj (hunk 2 above)
Comment (waynemunro) [reply test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj]: done

File: src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj (hunk 3 above)
Comment (waynemunro) [reply src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj]: Reverted as requested.

File: ServiceFabric.Mocks.sln [hunk 5]
```diff
@@ -24,13 +24,15 @@ Global
 		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{2832EB4C-6B37-4CEA-9F8B-7896757E69A7}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{2832EB4C-6B37-4CEA-9F8B-7896757E69A7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (loekd) [inline ServiceFabric.Mocks.sln:27]: please revert these changes too (also about x64)

File: test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj [hunk 6]
```diff
@@ -20,7 +20,7 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <Platforms>x64</Platforms>
+    <Platforms>AnyCPU</Platforms>
```
Comment (loekd) [inline test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj:23]: please revert these changes too (also about x64)

**Review by loekd — CHANGES_REQUESTED** [review]

**Comment by waynemunro** [comment]
Type in commit comments. Correction.
Revert to x64 and removed Any CPU

File: test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj (hunk 6 above)
Comment (waynemunro) [reply test/ServiceFabric.Mocks.NetCoreTests/ServiceFabric.Mocks.NetCoreTests.csproj]: done, thanks

**Comment by loekd** [comment]
The package 6.0.4 is now available. (the version wasn't bumped in the csproj, fixed that just now)

**Comment by waynemunro** [comment]
@loekd when will the updated nuget version be available?
